decidim-core/app/views/decidim/homepage/show.html.erb

```diff
@@ -10,5 +10,6 @@
 <% Decidim::ContentBlock.published.for_scope(:homepage, organization: current_organization).each do |content_block| %>
   <% next unless content_block.manifest %>
 
+  <a id="<%= content_block.manifest.name %>"></a>
```

Comment:
Oh, I didn't catch how the HTML looks like. 

We pass from this: 

![image](https://user-images.githubusercontent.com/717367/151788188-3c17bf6b-1407-42fb-a4f4-8dfe88bfbd96.png)

To this: 

![image](https://user-images.githubusercontent.com/717367/151788057-a6a0b4ef-9ef6-4f33-99a4-a945a8d8deaa.png)


For me, it's much cleaner the before than the after. Can we move every `id` attribute to the section itself? I see that, as a matter of fact, we're already doing that in the `statistics` section. Sorry that I didn't catch it before.

Reply:
@andreslucena I have implemented like : 
![image](https://user-images.githubusercontent.com/105683/151932948-c07b0b93-0220-4be5-9840-25f8eee972a2.png)